decidim-meetings/app/views/decidim/meetings/meetings/_meetings.html.erb

```diff
@@ -1,5 +1,17 @@
+  <% if @meetings_warning %>
```

Comment:
@oriolgual I think it's useful to have two different warnings. If we change the behavior on load time (past instead of upcoming) it's nice to show a warning to the user.